button-inc/emissions-elt-demo
# fix: gcp cloud build does not pass

Addresses #199. This PR allows the Nextjs site to build within GCP Cloud Build. 

⚠️: More work is required to get the site fully operational, see #226 for the errors related to auth.

## Changes

- Removed unused pages with errors (see [the discussion in the bug report](https://github.com/button-inc/emissions-elt-demo/issues/199#issuecomment-1430411876)).
- Fixed some minor typescript & linting errors for the build to pass.
- Added a Telus package and some peer dependencies required by the Telus UI elements.
- Added secrets required by NextAuth to GCP secret manager and passed them through Docker/Cloud Build scripts to make them available. 

## To test

- Check the Cloud Build log for the latest build [722c22d4](https://console.cloud.google.com/cloud-build/builds;region=global/722c22d4-06c2-4d5c-ad5f-8dcdc77e1eea?project=emissions-elt-demo).
- Check the live site displays at [34.125.36.177](http://34.125.36.177/).
